recipes/default.rb

```diff
@@ -15,3 +15,30 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+include_recipe 'osl-docker'
+
+# Synapse management user
+user 'synapse-host' do
```

Comment:
Any reason why this can't be the following?
```suggestion
user 'synapse' do
```